### PR TITLE
Expand journal-cms database space

### PR DESCRIPTION
**Pull request by giorgiosironi**
For the incident in https://elifesciences.slack.com/archives/C6N559E2F/p1569922331004400

### CODE REVIEW

**Comment by giorgiosironi** [comment]
Since the disk is full, have to do this through the [console first](https://console.aws.amazon.com/cloudformation/home?region=us-east-1#/stacks/events?filteringText=journal-cms--prod&filteringStatus=active&viewNested=true&hideStacks=false&filter=active&stackId=arn%3Aaws%3Acloudformation%3Aus-east-1%3A512686554592%3Astack%2Fjournal-cms--prod%2F39a4a330-15be-11e7-b853-50d5cad95262).

File: projects/elife.yaml [hunk 1]
```diff
@@ -988,21 +988,21 @@ journal-cms:
...
                 type: db.t2.medium
                 engine: MySQL
-                version: '5.7'
+                version: '5.7.23'
                 multi-az: true
-                storage: 24 # GB
+                storage: 48 # GB
             ext:
-                size: 30 # GB
+                size: 60 # GB
```
Comment (giorgiosironi) [inline projects/elife.yaml]: accidentally increased this as well, and now can't go back

File: projects/elife.yaml [hunk 2]
```diff
@@ -988,21 +988,21 @@ journal-cms:
             rds:
                 type: db.t2.small
                 engine: MySQL
-                version: '5.7'
-                storage: 24 # GB
+                version: '5.7.23'
```
Comment (giorgiosironi) [inline projects/elife.yaml]: `continuumtest` and `prod` instances were stuck on `5.7.22` and `5.7.19` respectively, despite minor version upgrades being turned on. Need a AWS ticket for this to understand what's wrong

File: projects/elife.yaml (hunk 1 above)
Comment (lsh-0) [reply projects/elife.yaml]: that's fine.

if the database is too large for 30GB it has problems regardless.

File: projects/elife.yaml (hunk 2 above)
Comment (lsh-0) [reply projects/elife.yaml]: Ticket has been opened. I've gone through the docs and can't find any reason why it shouldn't have happened or weasel words or exceptions to the rule.

There are AWS forum posts and Stackoverflow posts about this as well with no resolution besides simply doing it yourself.

Comment (giorgiosironi) [reply projects/elife.yaml]: Where's the `:grrr:` emoji when you need it

**Comment by giorgiosironi** [comment]
`continuumtest` is up to date, applying to `prod`.

**Comment by giorgiosironi** [comment]
Applied to `prod`, `end2end` has not been touched and remains as the example of the minor version upgrade problem.

**Pull request merged**